kernel/nsh/builtins/info.c

```diff
@@ -0,0 +1,126 @@
+// SPDX-FileCopyrightText: CGL-KFS
+// SPDX-License-Identifier: BSD-3-Clause
+
+/* kernel/nsh/builtins/info.c
+ *
+ * Insert file description here
+ *
+ * created: 2022/12/09 - mrxx0 <chcoutur@student.42.fr>
+ * updated: 2022/12/12 - xlmod <glafond-@student.42.fr>
+ */ 
+
+#include <stdint.h>
+#include <kernel/print.h>
+#include <kernel/string.h>
+#include <kernel/kpm.h>
+
+#include "../../gdt_internal.h"
+#include "../../idt_internal.h"
+#include "registers_internal.h"
+
+extern t_gdt_entry *gdt;
+extern t_gdt_ptr gdtp;
+extern uint32_t stack_top;
+extern uint32_t stack_bottom;
+extern buddy_t *buddy;
+extern t_idt_entry idt[256];
+extern t_idt_ptr idtp;
+
+#define PRINT(reg, var, format) \
+	reg(var); kprintf(format, var)
+
+void info_registers()
+{
```

Comment:
Braces should be placed at the end of the function name. (Samething for other functions)

Reply:
this file should not be here, i will remove it.